src/core/hero/hero.module.scss

```diff
@@ -38,6 +31,10 @@
     }
 
     .link {
+      position: absolute;
+      left: 32px;
```

Comment:
Not a big deal but `var(--spacing-l)` and `var(--spacing-4-xl)` could be used for 32px and 64px

Reply:
yes, thanks, i ll fix